src/controller/index.ts

```diff
@@ -352,7 +350,8 @@ export default class SimulariumController {
     public changeFile(
         connectionParams: SimulatorConnectionParams,
         // TODO: push newFileName into connectionParams
-        newFileName: string
+        newFileName: string,
+        keepRemoteConnection = false
```

Comment:
Did you consider making this a member variable that's more persistent, like a mode or state of the controller, as opposed to passing it in all the time?

Reply:
I just re-read the PR text and it makes sense to me how you did it. If you imagine some kind of loadbalancer in place then it might make sense to always have to reconnect.